sdk/template/template/test/public/configurationClient.spec.ts

```diff
@@ -80,25 +80,20 @@ describe("[AAD] ConfigurationClient functional tests", function () {
   });
 
   describe("#getConfigurationSetting", () => {
-    // TODO: This test is skipped on macOS pending investigation (see https://github.com/Azure/azure-sdk-for-js/issues/31296)
-    it.skipIf(typeof process !== "undefined" && process.platform === "darwin")(
-      "predetermined setting has expected value",
-      async () => {
-        const key = assertEnvironmentVariable("APPCONFIG_TEST_SETTING_KEY");
-        const expectedValue = assertEnvironmentVariable("APPCONFIG_TEST_SETTING_EXPECTED_VALUE");
+    it("predetermined setting has expected value", { timeout: 50000, retry: 3 }, async () => {
```

Comment:
maybe a smaller value like 20 * 1000? The passing tests took 5-ish seconds to run.

Reply:
I haven't studied it enough in CI to determine but we should have an investigation thoroughly of CI before we lower it.  We should have some sort of follow up item on why the test-recorder has perf issues.